share/extender/build_input.yml

```diff
@@ -395,12 +395,12 @@ platforms:
   web:
     # It's here to whitelist this as a platform, currently used for general settings
     env:
-        EM_CACHE:               "{{env.EMSCRIPTEN_CACHE_3_1_55}}"
-        EM_CONFIG:              "{{env.EMSCRIPTEN_CONFIG_3_1_55}}"
-        EMSDK_PYTHON:           "{{env.EMSCRIPTEN_PYTHON_3_1_55}}"
-        PATH:                   "{{env.EMSCRIPTEN_PATH_3_1_55}}:{{env.PATH}}"
-        EMSCRIPTEN_HOME:        "{{env.EMSCRIPTEN_HOME_3_1_55}}"
-        EMSCRIPTEN_BIN:         "{{env.EMSCRIPTEN_BIN_3_1_55}}"
+        EM_CACHE:               "{{env.EMSCRIPTEN_CACHE_3_1_65}}"
+        EM_CONFIG:              "{{env.EMSCRIPTEN_CONFIG_3_1_65}}"
+        EMSDK_PYTHON:           "{{env.EMSCRIPTEN_PYTHON_3_1_65}}"
+        PATH:                   "{{env.EMSCRIPTEN_PATH_3_1_65}}:{{env.PATH}}"
+        EMSCRIPTEN_HOME:        "{{env.EMSCRIPTEN_HOME_3_1_65}}"
+        EMSCRIPTEN_BIN:         "{{env.EMSCRIPTEN_BIN_3_1_65}}"
```

Comment:
Relevant to the discussion in our slack channel.
In the future, we should remove the version numbers from these variables, as it'll be one file less to update/remember when updating sdk versions.